pages/docs/how-to-guides/bio.mdx

```diff
@@ -25,5 +25,3 @@ _If you need help on how to edit this file, please see the [Editing guide](/docs
 3. Now you can commit your file and create a Pull Request, for more details please see [Editing guide](/docs/how-to-guides/editing)
 
 _Clicks on these social shortcuts that match any of your links will also increment their respective counters_
-
-export default ({ children }) => <DocsLayout>{children}</DocsLayout>;
```

Comment:
This line is required for the `mdx` file to load the docs layout - it is not displayed on the rendered page https://linkfree.eddiehub.io/docs/how-to-guides/bio